src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+
+  &__star {
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:last-child {
+    background-image: url(../images/star.svg);
```

Comment:
Ensure that the path `../images/star.svg` is correct relative to the location of the compiled CSS file. If the path is incorrect, adjust it accordingly.